activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -93,6 +99,34 @@ def travel(duration, &block)
       #   end
       #   Time.current # => Sat, 09 Nov 2013 15:34:49 EST -05:00
       def travel_to(date_or_time)
+        if block_given? && simple_stubs.stubbed?(Time, :now)
+          travel_to_nested_block_call = <<-MSG
+
+      Calling `travel_to` with a block, when we have previously already made a call to `travel_to`, can lead to confusing time stubbing.
+
+      Instead of:
+
+         travel_to 2.days.from_now do
+           # 2 days from today
+           travel_to 3.days.from_now do
+             # 5 days from today
+           end
+         end
+
+      preferred way to achieve above is:
+
+         travel_to 2.days.from_now do
+           # 2 days from today
+         end
+
+         travel_to 5.days.from_now do
```

Comment:
My point is that our API for relative times is `travel` and not `travel_to`.

> The first one reads out in a more natural way.

Subjective. I personally find the second one more natural. But that's why we have both for the times when one reads better than the other 😁